### PR TITLE
set exit code

### PR DESCRIPTION
**Pull request by nritsche**

### CODE REVIEW

File: scripts/coco [hunk 1]
```diff
@@ -214,11 +214,12 @@ if hasattr(parsed_args, "func"):
                     parsed_args.style, STYLES
                 )
             )
+            sys.exit(1)
```
Comment (anjakefala) [inline scripts/coco:217]: Why was this one added in this location with this `else` statement nested within `if result`?

File: scripts/coco [hunk 2]
```diff
@@ -197,7 +197,7 @@ if hasattr(parsed_args, "func"):
     if parsed_args.endpoint == "save-state":
         parsed_args.data["overwrite"] = parsed_args.overwrite
         del parsed_args.overwrite
-    result = parsed_args.func(
+    success, result = parsed_args.func(
```
Comment (anjakefala) [inline scripts/coco:200]: Is `success` used?

File: scripts/coco [hunk 3]
```diff
@@ -214,11 +214,12 @@ if hasattr(parsed_args, "func"):
...
     if not parsed_args.silent:
         print("Done.")
 else:
     parser.print_help()
     print("\ncoco: error: an endpoint must be specified")
-    exit(1)
+    sys.exit(1)
 
-exit(0)
+sys.exit(not success)
```
Comment (anjakefala) [inline scripts/coco:225]: where is `not success` defined?

**Review by anjakefala — CHANGES_REQUESTED** [review]
I have questions! =)

File: scripts/coco (hunk 3 above)
Comment (nritsche) [reply scripts/coco:225]: `not` is a logical negation, `success` is defined above

File: scripts/coco (hunk 1 above)
Comment (nritsche) [reply scripts/coco:217]: This branch is for "Unknown print style", as the print statement says. It was previously continueing to print "Done" and exit(0), but it's an error really.

File: scripts/coco (hunk 3 above)
Comment (anjakefala) [reply scripts/coco:225]: Wow, syntax highlighting is my friend.

Thank you!

**Review by anjakefala — APPROVED** [review]
Awesome!

**Pull request merged**